utils/utils.go

```diff
@@ -26,6 +26,7 @@ func RenderMarkdown(text string) (string, error) {
 	if isColorEnabled() {
 		style = "dark"
 	}
+	text = strings.Map(filterCarriageReturn, text)
```

Comment:
Thanks! This does the trick, but I wonder can this be more idiomatic and use ReplaceAll to explicitly replace `\r\n` with just `\n`?

```go
// ensure no carriage return characters are present in output
text = strings.ReplaceAll(text, "\r\n", "\n")
```

It should accomplish the same, but be a more clearer as to why we are doing this for someone who will be reading this code.

Reply:
Okay, Didn't figure it out this way. I'll update the same.